app/program/page.tsx

```diff
@@ -54,7 +54,7 @@ const Page = () => {
             <TalkInfoCard
               talk={{
                 title: "Introduction - cyntx",
-                description: "",
+                description: "Cytnx: An Introduction to the Tensor Network Library for Quantum Physics",
                 room: "R716",
```

Comment:
Could you please also add the speaker name (`Ying-Jer Kao`) for Cytnx talk ?

Reply:
Done !